src/@batch-flask/ui/property-list/table-property/table-property.html

```diff
@@ -13,8 +13,20 @@
             </tr>
         </table>
         <div class="expand-container" *ngIf="canExpand || canCollapse">
-            <bl-button type="wide" color="light" class="expand" *ngIf="canExpand" (click)="expanded = true" aria-expanded="false">Show {{rows.length - defaultDisplay}} more</bl-button>
-            <bl-button type="round" color="light" class="collapse" *ngIf="canCollapse" (click)="expanded = false" aria-expanded="true"><i class="fa fa-caret-up"></i></bl-button>
+            <bl-button
+                [type]="expanded ? 'round' : 'wide'"
+                color="light"
+                [attr.class]="expanded ? 'expand' : 'collapse'"
+                [attr.aria-expanded]="expanded"
+                [ngSwitch]="expanded"
+                (click)="toggleExpand($event)"
+                (keydown)="toggleExpand($event)"
+            >
```

Comment:
NIt: would it be better if we just moved this `>` behind `(keydown)` instead of its own line?